2017/JULY/JULY17/EXPTREE/EXPTREE.py

```diff
@@ -0,0 +1,52 @@
+#__BY:sirjan13__
+
+def gcd(n,m):
+    if n==0:
+        return m
+    return gcd(m%n,n)
+
+t1,t2=0,0
+
+def egcd(a, b):
+    x,y, u,v = 0,1, 1,0
+    while a != 0:
+        q, r = b//a, b%a
+        m, n = x-u*q, y-v*q
+        b,a, x,y, u,v = a,r, u,v, m,n
+    gc = b
+    return gc, x, y
+    
+def mi(a,m):
+    g = egcd(a,m)
+    ans = (g[1] %m + m )%m
+    return ans
+ 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpufcxdz6g/2017/JULY/JULY17/EXPTREE/EXPTREE.py
+++ b/tmp/tmpufcxdz6g/2017/JULY/JULY17/EXPTREE/EXPTREE.py
@@ -20,7 +20,7 @@
     g = egcd(a,m)
     ans = (g[1] %m + m )%m
     return ans
- 
+
 mod1 = 10**9+7
 mod2 = mod1 + 2
 for cppp in range(int(raw_input())):
```